signale.js

```diff
@@ -80,7 +81,7 @@ class Signale {
   }
 
   _log(message) {
-    process.stdout.write(message + '\n');
+    process.stdout.write(message + os.EOL);
```

Comment:
Windows handles `\n` fine for terminal output. Even Node.js uses only `\n` for `console.log`: https://github.com/nodejs/node/blob/fa0cac51b9d30c5ce3a1f82aba02754be447ae65/lib/console.js#L141-L148